README.md

```diff
@@ -874,14 +860,13 @@ Below you will find a diagram showing the microservice tech stack, both for the
   <img src="images/microservice-tech.jpeg" />
 </p>
 
-
-▶️ 𝐏𝐫𝐞-𝐏𝐫𝐨𝐝𝐮𝐜𝐭𝐢𝐨𝐧
+#### Pre-production
```

Comment:
I don't think we want to change these into sub-headings. It makes the outline view super busy:

<img width="313" alt="image" src="https://github.com/ByteByteGoHq/system-design-101/assets/50348/127ee173-b32e-4d0a-93d9-79d58afcf1b4">